lib/cisco_node_utils/bgp.rb

```diff
@@ -531,6 +527,35 @@ def default_reconnect_interval
       config_get_default('bgp', 'reconnect_interval')
     end
 
+    # Route Distinguisher (Getter/Setter/Default)
```

Comment:
Sorry - you are correct that would have been better.

To mitigate this I've gone through and marked my changes. Only exception is the test_ file which has a lot of changes - but mainly cleanup and fluff.

Is that ok?